src/PgManyToManyRelationPlugin.js

```diff
@@ -52,23 +474,147 @@ module.exports = function PgManyToManyRelationPlugin(
             .join("-and-")}-list`
         );
       },
+      edgeViaJunction(nodeTypeName, junctionTypeName) {
+        return this.upperCamelCase(
+          `${this.pluralize(nodeTypeName)}-via-${junctionTypeName}-edge`
+        );
+      },
+      connectionViaJunction(nodeTypeName, junctionTypeName) {
+        return this.upperCamelCase(
+          `${this.pluralize(nodeTypeName)}-via-${junctionTypeName}-connection`
+        );
+      },
+      // to make the tests pass
+      builtin: inflection.builtin || (name => name),
```

Comment:
Do you know why this function is necessary for the tests to pass? I would expect `inflection.builtin` to be already defined.

Reply:
Seem to be testing against an older engine version (4.1.0). builtin was added in 4.3.1: https://github.com/graphile/postgraphile/releases/tag/v4.3.1

Reply:
Good call. @singingwolfboy, feel free to update to 4.3.1. That was released more than six months ago so I'm fine with that being the new minimum version.